code/src/index.css

```diff
@@ -1,13 +1,182 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+  font-family: 'PT Mono', monospace;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: #ffd5d5;
+}
+
+.container {
+  margin: 0 auto;
+  width: 320px;
 }
 
 code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
+  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
+h1 {
+  font-size: 16px;
+}
+
+label {
+  margin: 10px 0px;
+}
+
+.date {
+  font-style: italic;
+  font-size: 14px;
+}
+
+textarea {
+  resize: none;
+}
+
+.main {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #000;
+  width: 280px;
+  padding: 8px;
+  box-shadow: 5px 5px #000;
+  background-color: #f2f0f0;
+  margin-top: 20px;
+}
+
+.message-container {
+  width: 280px;
+  border: 1px solid #000;
+  margin: 20px 0px;
+  padding: 8px;
+  box-shadow: 5px 5px #000;
+  background-color: #fff;
+  overflow-wrap: break-word;
+}
+
+.disabled-textarea {
+  color: red;
+}
+
+.characters-left {
+  font-size: 15px;
+}
+
+.submit-button {
+  border-radius: 20px;
+  border: none;
+  background-color: #ffadad;
+  font-family: 'PT Mono', monospace;
+  color: #000;
+  width: 70%;
+  padding: 4px 0px;
+  margin: 10px 0px;
+  display: flex;
+  justify-content: space-evenly;
+}
+
+.submit-text {
+  margin: 0;
+}
```

Comment:
Super small detail, but if you want the text aligned with the hearts you could put align-items: center in the submit button.